enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -462,27 +462,40 @@ compatibility, the Ingress Operator must do one of the following:
 Placement, possible migration, and ownership of the SMCP need to be clarified in
 this enhancement.  This is an open design question.
 
-#### Managing the ServiceMeshMemberRoll
+#### Managing Istiod's Control Plane Scope
 
-OSSM 2.3 only watches resources in namespaces specified in an
-ServiceMeshMemberRoll CR associated with the ServiceMeshControlPlane; see
-[OSSM-924](https://issues.redhat.com/browse/OSSM-924).  We hope that this limitation will be removed from OSSM in time
-for this enhancement; this is an open release-planning question.  Otherwise, the
-Ingress Operator or cluster admin must add each namespace with HTTPRoute objects
-that Istio should reconcile to the ServiceMeshMemberRoll, and the approach we
-take and ownership of ServiceMeshMemberRoll need to be clarified in this
-enhancement; this is an open design question.
+OSSM has traditionally only watched resources in namespaces specified in an
+ServiceMeshMemberRoll CR associated with the ServiceMeshControlPlane; however,
+OSSM recently added support for a cluster-scoped control plane mode in [OSSM-1320](https://issues.redhat.com/browse/OSSM-1320).
+In cluster-scoped mode, Istiod watches all namespaces so that Gateway API objects
+will be reconciled in any namespaces, removing the need for ServiceMeshMemberRolls.
 
-#### Gateway API v1alpha2 CRDs
+In OSSM 2.3, cluster-scoped mode is turned on in the SMCP via the unstructured
```

Comment:
I realize https://github.com/openshift/enhancements/pull/1332/files#diff-7973eca3c2f112ece8f8585f4f4c6567fc1c1b4fa95be7f88a27ce08820d89f7R321 has a little bit of the same wording, but maybe it's worth rehashing it in more detail here.

Reply:
nit: If it has to change again, you'll need to update this again.  It could be better to record this information elsewhere, as it's not really needed in the proposal, and the proposal is long already.